### PR TITLE
Use WalletStandard errors in examples

### PR DESCRIPTION
**Pull request by olistic**
This PR uses the new [standard errors](https://github.com/wallet-standard/wallet-standard/tree/6d2f68198284bd68470c1a88088f54707abf46dc/packages/core/errors) in the examples.

### CODE REVIEW

**Comment by olistic** [comment]
@jordaaash @steveluscher is this still a valuable contribution?

**Comment by steveluscher** [comment]
Sounds great! Add these codes to https://github.com/wallet-standard/wallet-standard/tree/master/packages/core/errors and use them here.

**Review by steveluscher — CHANGES_REQUESTED** [review]
Sounds great! Add these codes to https://github.com/wallet-standard/wallet-standard/tree/master/packages/core/errors and use them here.

File: packages/core/errors/src/codes.ts [hunk 1]
```diff
@@ -29,6 +29,10 @@
 export const WALLET_STANDARD_ERROR__REGISTRY__WALLET_NOT_FOUND = 3834000 as const;
 export const WALLET_STANDARD_ERROR__REGISTRY__WALLET_ACCOUNT_NOT_FOUND = 3834001 as const;
 
+// User-related errors.
+// Reserve error codes in the range [4001000-4001999].
+export const WALLET_STANDARD_ERROR__USER__REQUEST_REJECTED = 4001000 as const;
```
Comment (olistic) [inline packages/core/errors/src/codes.ts:34]: @steveluscher this was originally 4001, inspired by [EIP-1193's errors](https://eips.ethereum.org/EIPS/eip-1193#provider-errors). Now it's completely arbitrary. Let me know if you want to change it.

**Comment by olistic** [comment]
> Sounds great! Add these codes to https://github.com/wallet-standard/wallet-standard/tree/master/packages/core/errors and use them here.

Done.

**Comment by steveluscher** [comment]
Added via Giphy

Did we lose the changes to `packages/example/dapp/src/pages/Connect.tsx` in your latest update? Feel free to add that back in a separate PR if you like.

**Pull request merged**